crypto/conf/conf_lib.c

```diff
@@ -239,6 +239,18 @@ int NCONF_load_bio(CONF *conf, BIO *bp, long *eline)
     return conf->meth->load_bio(conf, bp, eline);
 }
 
+static STACK_OF(CONF_VALUE) *_CONF_get_section_values(const CONF *conf,
+                                                      const char *section)
```

Comment:
This is called in exactly one place, and is very little code.  I see no reason to keep it as a separate function.

You might actually want to consider the same thing for the other `_CONF` functions, i.e. integrate their code in the spots where they are used, unless they are multiple spots.

Reply:
I wanted to make as few changes as possible, figuring it would ease the review (hah).  I inlined it.  I don't think there are others, but I will check.  Additional commit pushed.